itest/litd_custom_channels_test.go

```diff
@@ -405,6 +409,11 @@ func testCustomChannels(_ context.Context, net *NetworkHarness,
 	charlieAssetBalance -= paidAssetAmount
 	daveAssetBalance += paidAssetAmount
 
+	// We should also be able to do a multi-hop BTC only payment, paying an
+	// invoice from Erin by Charlie.
+	createAndPayNormalInvoiceWithBtc(t.t, charlie, erin, 2000)
+	logBalance(t.t, nodes, assetID, "after BTC only invoice")
```

Comment:
does this also assert that the balance is as expected?

Reply:
No, but we do that later. That's just for visual confirmation. And currently we don't track and/or assert the BTC balance. But since we don't really change any logic in the BTC-only flow, just making sure the payment arrives should be enough.